src/redis_cache.cc

```diff
@@ -118,6 +127,11 @@ RedisCache::Create(
   sw::redis::ConnectionOptions options;
   sw::redis::ConnectionPoolOptions poolOptions;
 
+  // try pulling user/password from environment fist
+  // override if present in the config
+  setOptionFromEnv(USERNAME_ENV_VAR_NAME, options.user);
```

Comment:
Might as well just add in TLS cert/pem file location as option and support TLS communication? SSL?

Reply:
Opened #7 to address just this, not sure they really need to be accessible in the env vars, doing that for user/password as those credentials can be sensitive, for TLS you're really just passing in file names.